event/test/testCocktailGenerator.cpp

```diff
@@ -0,0 +1,44 @@
+#include <iostream>
+#include "CocktailGenerator.h"
+#include "Event.h"
+#include "Track.h"
+#include "Particle.h"
+#include "vectorFlow/PhysicalConstants.h"
+#include "vectorFlow/SystemOfUnits.h"
+#include "base/Vector3D.h"
+
+#define N_EVENTS 10
+
+int main(void) {
+    // Add CocktailGenerator
+    vecgeom::Vector3D<double> vertex(0., 0., 10.);
+    CocktailGenerator* CG = new CocktailGenerator(vertex);
```

Comment:
Cleanup CG at the end (or create/handle on the stack)

Reply:
Fixed. Changed "CG" (now "cocktailGen") to be managed in the stack rather than the heap.